src/uwtools/tests/drivers/test_ioda.py

```diff
@@ -0,0 +1,112 @@
+# pylint: disable=missing-function-docstring,protected-access,redefined-outer-name
+"""
+IODA driver tests.
+"""
+import datetime as dt
+from unittest.mock import DEFAULT as D
+from unittest.mock import patch
+
+from pytest import fixture
+
+from uwtools.drivers.driver import Driver
+from uwtools.drivers.ioda import IODA
+
+# Fixtures
+
+
+@fixture
+def config(tmp_path):
+    base_file = tmp_path / "base.yaml"
+    base_file.write_text("foo: bar")
+    return {
+        "ioda": {
+            "execution": {
+                "batchargs": {
+                    "export": "NONE",
+                    "cores": 1,
+                    "stdout": "/path/to/file",
+                    "walltime": "00:02:00",
+                },
+                "envcmds": [
+                    "module load some-module",
+                    "module load jedi-module",
+                ],
+                "executable": "/path/to/bufr2ioda.x",
+                "mpiargs": ["--export=ALL", "--ntasks $SLURM_CPUS_ON_NODE"],
+                "mpicmd": "srun",
```

Comment:
I think this should be removed as well if it needs serial execution? After working with `ioda` did you find that it didn't need serial execution? Or should it follow the `jedi` driver because we are calling it form the `ioda` driver class?